tests/test_version.py

```diff
@@ -4,17 +4,19 @@
 
 import pytest
 import os
+import nimbleclient
```

Comment:
although this won't create a problem. but can we replace this as below
import nimbleclient => imports everything inside nimbleclient
from nimbleclient import __init__  ..==> just imports init

Reply:
note that you will also need to modify 
` with open(nimbleclient.__file__) as fh:`  and replace with  `with open(__init__.__file__) as fh:`